modules/ROOT/pages/apikit-4-soap-fault-task.adoc

```diff
@@ -3,12 +3,12 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can use SoapUI to identify an operation you need to implement by making the API fail. You use the `OrderTshirt` operation to create a request in SOAP UI to place an order for a large t-shirt for John. The API cannot handle this operation and a SOAP fault occurs.
+To know what happens with an implemented operation, you make the API fail using SoapUI. Take the steps of the following example to create a request in SoapUI using the `OrderTshirt` operation to place an order for a large t-shirt for John. Because the API cannot handle this operation, a SOAP fault occurs.
```

Comment:
```suggestion
To know what happens with an implemented operation, make the API fail using SoapUI. Take the steps of the following example to create a request in SoapUI using the `OrderTshirt` operation to place an order for a large t-shirt for John. Because the API cannot handle this operation, a SOAP fault occurs.
```

Reply:
"Take the steps" is not the right wording. I know "Follow the steps in the following example..." has "follow" twice, but it is what you want the users to do.

Reply:
OK! Thanks!